reactor-netty-core/src/main/java/reactor/netty/resources/LoopResources.java

```diff
@@ -43,11 +50,12 @@ public interface LoopResources extends Disposable {
 			"" + Math.max(Runtime.getRuntime().availableProcessors(), 4)));
 
 	/**
-	 * Default selector thread count, fallback to -1 (no selector thread).
+	 * Default selector thread count, fallback to -1 (no selector thread),
+	 * but fallback to 1 if pending task logging is enabled.
 	 */
 	int DEFAULT_IO_SELECT_COUNT = Integer.parseInt(System.getProperty(
 			ReactorNetty.IO_SELECT_COUNT,
-			"" + -1));
+			DEFAULT_PENDING_TASK_LOG_THRESHOLD > -1 ? "" + 1 : "" + -1));
```

Comment:
If SelectLoop is not enabled, the number of pending tasks may be distorted (e.g., when sleeping in a code block).